plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go

```diff
@@ -57,6 +57,12 @@ func ComputeTaintTolerationPriorityMap(pod *v1.Pod, meta interface{}, nodeInfo *
 	if node == nil {
 		return schedulerapi.HostPriority{}, fmt.Errorf("node not found")
 	}
+	// get taints from node info cache.
+	taints, err := nodeInfo.Taints()
+	if err != nil {
+		// fallback to computing taints from node Spec
+		taints = node.Spec.Taints
```

Comment:
Local clusters will create all the same components that scaled ones create.  What about this improvement makes it only visible in distributed clusters?  The calls to the client-cache will, after all, be reading locally, and i don't see any calls to the API server being reduced here?